index.js

```diff
@@ -97,10 +97,28 @@ function filter (obj, { protoAction = 'error', constructorAction = 'error', safe
   return obj
 }
 
+function parse (text, reviver, options) {
+  const stackTraceLimit = Error.stackTraceLimit
+  try {
+    Error.stackTraceLimit = 0
+    const result = _parse(text, reviver, options)
+    Error.stackTraceLimit = stackTraceLimit
+    return result
+  } catch (e) {
+    Error.stackTraceLimit = stackTraceLimit
```

Comment:
Why not put this in a `finally` block?

Reply:
Is it safe? I mean, I interupt the flow by throwing. How is it ensured, that finally is called before the next command after the parse command?

```js
function test() {
  try {
    throw new Error()
  } catch (e) {
    console.log('catch')
    throw new Error()
  } finally {
    console.log('finally')
  }
}

try {
  test()
} catch (e) {}

console.log('done')
```

I get:

```sh
catch
finally
done
```

so it seems valid. Is there a rule in js, that finally always is handled, before a throw or return interrupts the flow?

Reply:
nice, thank you. 

It also improves the performance.